src/main/java/com/faforever/client/preferences/ui/SettingsController.java

```diff
@@ -219,6 +232,24 @@ public void initialize() {
     passwordChangeErrorLabel.setVisible(false);
   }
 
+  private void configureTimeSetting(Preferences preferences) {
+    timeComboBox.setItems(FXCollections.observableArrayList(TimeInfo.getAllDisplayNames()));
+    timeComboBox.setOnAction(event -> {
+      newTimeFormatSelected(event);
+    });
+    timeComboBox.setDisable(false);
+    timeComboBox.setFocusTraversable(true);
+    timeComboBox.getSelectionModel().select(TimeInfo.getIndexByDisplayName(preferences.getChat().getUkTime()));
+  }
+
+  private void newTimeFormatSelected(Event event) {
+    logger.info("newTimeFormat is " + timeComboBox.getValue().toString());
```

Comment:
1. Don't log this to `info` (see [logging guidelines](https://github.com/FAForever/downlords-faf-client/wiki/Contribution-guidelines#logging))
2. `.toString()` is not necessary since the `+` operator calls this implicitly
3. The `+` operator shouldn't be used for logging with slf4j because it always causes `toString()` to be called, even if the message is never logged due to a higher log level setting (`toString()` can be expensive, especially in loops)
4. Log messages should be as clean as text on the UI

`logger.debug("New time format is: {}", timeCombobox.getValue);`